README.md

```diff
@@ -32,4 +32,4 @@ $ foreman start
 
 The original [Hemingway App](http://www.hemingwayapp.com/).
 
-Photo by [Leon Bliss](https://www.flickr.com/photos/leonbiss/) via [Unsplash](http://unsplash.com/).
\ No newline at end of file
+Photo by [Leon Bliss](https://www.flickr.com/photos/leonbiss/) via [Unsplash](http://unsplash.com/).
```

Comment:
What is this change? It looks like an issue with your text editor that you mentioned before, but the lines look identical to me.